tests/simple/testdata/comparisons.textproto

```diff
@@ -75,12 +75,6 @@ section {
     expr: "'Am\\u00E9lie' == 'Ame\\u0301lie'"
     value: { bool_value: false }
   }
-  test {
-    name: "no_string_normalization_surrogate"
-    description: "Should not replace surrogate pairs."
-    expr: "'\\U0001F436' == '\\uD83D\\uDC36'"
-    value: { bool_value: false }
-  }
```

Comment:
What's the purpose behind deleting this test? The idea is that we want to showcase that normalization is not being performed as part of CEL core.

Reply:
This unfortunately will only work as expected if the implementation represents strings using UTF-8...

It will give a false positive for other implementations, which will allow them to pass.

Reply:
Thanks for the catch.